components/nexusoperations/frontend/fx.go

```diff
@@ -28,6 +28,13 @@ func ConfigProvider(coll *dynamicconfig.Collection) *Config {
 		PayloadSizeLimit:              dynamicconfig.BlobSizeLimitError.Get(coll),
 		ForwardingEnabledForNamespace: dynamicconfig.EnableNamespaceNotActiveAutoForwarding.Get(coll),
 		MaxOperationTokenLength:       nexusoperations.MaxOperationTokenLength.Get(coll),
+		MetricTagConfig: dynamicconfig.NewGlobalCachedTypedValue(
```

Comment:
I'm going to remove this soon (finally). If #7052 lands first then you can just get rid of this and use the setting directly. If this lands first, I'll do it in that PR.